### ISSUE TITLE
Hamburger menu does not announce open/close to screenreader users

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#29]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
[Developer Portal van de Nederlandse Overheid](https://don.apps.digilab.network/)
[Aan de slag met API's](https://don.apps.digilab.network/kennisbank/apis/aan-de-slag/)

**Probleembeschrijving:**
De screenreader kondigt niet aan wanneer het hamburger menu opent of sluit, ook al heeft het een aria-expanded attribuut.

**Referentiecode:**
`<button aria-label="Navigatiebalk schakelen" aria-expanded="false" class="navbar__toggle clean-btn" type="button">`

**Gerelateerde WCAG:**
WCAG 4.1.2 Name, Role, Value

**Referentieafbeelding:**
<img width="1762" alt="An open menu with an aria-expanded attribute is not announced as open to screenreader users" src="https://github.com/user-attachments/assets/41d7d3f3-2c05-4b7d-a930-a68cae19387b" />

_**Referentienummer:** A014 + A003_
